Source/WebKit/Platform/IPC/StreamConnectionBuffer.h

```diff
@@ -122,12 +122,23 @@ class StreamConnectionBuffer {
 private:
     StreamConnectionBuffer(Ref<WebKit::SharedMemory>&&);
 
+#if CPU(ARM64)
+#define HEADER_POINTER_ALIGNMENT alignas(128)
+#elif CPU(X86_64)
+#define HEADER_POINTER_ALIGNMENT alignas(64)
+#else
+#define HEADER_POINTER_ALIGNMENT /* unknown */
+#endif
+
     struct Header {
         Atomic<ServerOffset> serverOffset;
         // Padding so that the variables mostly accessed by different processes do not share a cache line.
         // This is an attempt to avoid cache-line induced reduction of parallel access.
-        alignas(sizeof(uint64_t[2])) Atomic<ClientOffset> clientOffset;
+        HEADER_POINTER_ALIGNMENT Atomic<ClientOffset> clientOffset;
```

Comment:
Yeah,  128bits vs 128bytes... 

I think you could just hardcode it to max of for all supported platforms.

Reply:
OK.